### PR TITLE
[#13493] Can not switch to xDai chain

### LINKED ISSUE #13493: Can not switch to xDai chain
# Bug Report

## Problem 
The xDai now is[ Gnosis network ](https://www.anyblockanalytics.com/networks/xdai/) and an attempt to switch to the xDai chain got an error  'Network failed'
Blocker for part of e2e: 695890

#### Expected behavior
can switch to Gnosis network

#### Actual behavior
IMAGE 2022-06-16 12:31:38

### Reproduction

1) Open Status
2) Go to Advanced > Network 
3) switch to xDai

### Additional Information

- Status version: nightly 16/06/2021
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #13493

### CODE REVIEW

**Review by cammellos — APPROVED** [review]

**Comment by pavloburykh** [comment]
@flexsurfer thanx for the PR. Please, take a look at following issues

### ISSUE 1 Network fee is not being calculated during outgoing transaction.

Eventually - user is unable to send transaction

**Steps:** 

1. Login/create multiaccount
2. Switch to xDai Chain network
3. Get some xDai for tetsing (for example here https://www.gimlu.com/faucet )
4. Try to send some xDai

**Actual result:** Network fee is not calculated (endless spinner). Transaction can not be signed.

**Expected result:** Network fee is calculated. Transaction can be signed and sent. 


https://user-images.githubusercontent.com/97245802/174847785-17d5235b-e879-4ac4-9224-4c9f492a9f4e.mp4

[geth.log](https://github.com/status-im/status-react/files/8950631/geth.log)


### ISSUE 2 Transfers history could not be updated on xDai Chain network

@flexsurfer Not sure if this is a bug or expected behaviour. 

**Steps:** 

1. Create/recover multiaccount
6. Switch network to xDai Chain
7. Open wallet tab
8. Enter an account 
9. Wait until fetching history is processed
10. Observe an error

**Actual result:**  Transaction history is not fetched. Endless loader on History tab. Warning "Transaction history could not be updated. Check your connection and pull down to try again". 

**Expected result:** transaction history is fetched. 


https://user-images.githubusercontent.com/97245802/174848683-9ab9857b-be87-4c17-ab01-39a8fe47d81d.mp4

[geth.log](https://github.com/status-im/status-react/files/8950663/geth.log)

**Comment by flexsurfer** [comment]
it seems we can't use `https://rpc.ankr.com/gnosis`

**Comment by flexsurfer** [comment]
so we need to find another public rpc

**Comment by hesterbruikman** [comment]
@flexsurfer have you looked into other public rpc's?

> it seems we can't use https://rpc.ankr.com/gnosis
Any thoughts on why this one doesn't work?

I ask because there's a user report on the issue on our Community Discord https://discord.com/channels/641364059387854899/663386050625798184/991268494442242049

**Comment by flexsurfer** [comment]
@hesterbruikman i didn't find any :(

**Comment by hesterbruikman** [comment]
> @hesterbruikman i didn't find any :(

There's this one, I'm not sure if it can be trusted: https://rpc.gnosischain.com (seems legit, Nethermind)

Some other investigation:
- MetaMask support points to https://rpc.xdaichain.com/ as well, but it's not working
- xDAI is supported by POKT; perhaps we can ask them for an endpoint https://docs.pokt.network/home/supported-blockchains

I'll see if I can find out with the Gnosis team if they can point us to another public endpoint

**Comment by hesterbruikman** [comment]
@flexsurfer was pointed here on the Gnosis Chain Discord: https://twitter.com/gnosischain/status/1536766216689287168

TL;DR Endpoint from Pokt seems like the best option. @guylouis are you in touch with anyone at Pokt we can ask about a Gnosis Chain endpoint?

**Comment by flexsurfer** [comment]
https://gnosischain-rpc.gateway.pokt.network
https://xdai-rpc.gateway.pokt.network

dunno how these different , which one we should use

**Comment by hesterbruikman** [comment]
> https://gnosischain-rpc.gateway.pokt.network https://xdai-rpc.gateway.pokt.network
> 
> dunno how these different , which one we should use

Following [Pocket documentation](https://docs.pokt.network/home/use/public-rpc/gnosis-chain) it's the first: https://gnosischain-rpc.gateway.pokt.network/
